src/components/product-card/product-card.module.scss

```diff
@@ -1,3 +1,8 @@
+.productCard {
+    display: flex;
+    flex-direction: column;
```

Comment:
Note: this doesn't serve any practical purpose, this is just to have a class on the root element if users want to style it.